lib/ansible/module_utils/facts.py

```diff
@@ -261,9 +261,8 @@ def get_local_facts(self):
                 # try to read it as json first
                 # if that fails read it with ConfigParser
                 # if that fails, skip it
-                rc, out, err = self.module.run_command(fn)
                 try:
-                    out = out.decode('utf-8', 'strict')
+                    rc, out, err = self.module.run_command(fn, errors='strict')
```

Comment:
I could see a scenario where stdout is valid utf-8 that can be decoded with strict, but stderr could have garbage that would cause 'strict' to fail. Not sure if it's worth being able to specifying different error schemes for both (or different encodings for that matter)